### DISCUSSION TITLE
Where are they?

### DISCUSSION THREAD

**Discussion by nothingness6** [microsoft/WSL#11407] [Q&A]
I'm a total beginner of wsl and Linux. When I typed ls on wsl, it shows the list of the current directory. However, they are not existed anywhere. I opened the folder, C:\Program Files\WSL, but the folder structure is completely different. 
I even used an app, Everything, which can detect literally everything in your system. Typed "docker-desktop-user-distro", but there is no such a folder or file like that. 
Where are they? 
![wsl](https://github.com/microsoft/WSL/assets/105210258/06616f9c-ef17-436c-bf47-6bfd7763107f)

**Comment by Yewzir** [comment]
@nothingness6 

For me, that is `\\wsl.localhost\ubuntu` because my distro is Ubuntu. If yours is Ubuntu as well, that's the location.
Otherwise, you can also get there by just navigating to `\\wsl.localhost`, and proceed navigation from your distro folder.

Kind regards
Yewzir
